### ISSUE TITLE
Joining AdvRocketry dim makes client crash

### ISSUE THREAD

**Issue by krisjenner** [SpongePowered/SpongeForge#2525]
**I am currently running**

- SpongeForge version: 7.1.2
- Forge version: 2768
- Java version: _191
- Operating System: Winows10

All The Mods 3 Remix
Plugins (44): Minecraft, Minecraft Coder Pack, SpongeAPI, Sponge, ActiveTime, BetterChunkLoader, Broadcast, Buycraft, CarrotShop, CatClearLag, ClaimTweaks, CmdBuilder, CmdControl, CmdScheduler, CommandItems, CustomRestartMessage, DimReseter, Enjin Minecraft Plugin, GWMCrates, GWMLibrary, GriefPrevention, Holograms, HuskyCrates, HuskyUI, KeepInventorySponge, Laborus, Lottery, LuckPerms, MMCLogger, MMCReboot, MMCRestrict, MMCTickets, MagiBridge, Nucleus, PacketGate, ParticleEffectCombinator, PetBlocks, Prism, PrismPlus, Total Economy, TriviaQuest, UniversalMarket, Web-API, WorldEdit

Player crashes on attempt to join AdvRocketry "Moon" dimension (DIM-2)

<!-- Please provide a *full* server log (and crash-report if applicable).
[serverlog](https://gist.github.com/krisjenner/e885c423ca71ba39e5c63c812a01cc9e)
[crashreport](https://gist.github.com/krisjenner/30b5e4e021f949779cfbadc67bbcbdea)

**Comment by Keaton1188** [comment]
Have you tried resetting the DIM?

**Comment by fira** [comment]
Could be related to #2501

**Comment by krisjenner** [comment]
Yes tried resetting DIM it doesn't work updating to sponge 7.1.4 doesn't work either
